### DISCUSSION TITLE
How to use customAlphabet() with version 3?

### DISCUSSION THREAD

**Discussion by danielnordh** [ai/nanoid#424] [General]
Is it possible to use customAlphabet() with version 3?
I have a legacy codebase using Common JS so not able to use v4 at the moment.

Currently using:
`var { nanoid } = require("nanoid");`

But would like to use customAlphabet()

**Comment by ai** [comment]
```js
var { customAlphabet } = require("nanoid");

var nanoid = customAlphabet('1234567890abcdef', 10)
```
